lib/main.dart

```diff
@@ -305,6 +307,7 @@ class _TabPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    print("tab page reloading");
```

Comment:
I think this one should not be merged.

Reply:
Oh sorry, I was checking whether the page was reloading or not but I fixed it.